setup.py

```diff
@@ -0,0 +1,44 @@
+from setuptools import setup, find_packages
+from os.path import abspath, dirname, join as ospjoin
+import re
+
+here = abspath(dirname(__file__))
+
+
+def find_version(filename):
+    with open(filename, 'r') as f:
+        version_file = f.read()
+    version_match = re.search(r"^__version__ = ['\"]([^'\"]*)['\"]",
+                              version_file, re.M)
+    if version_match:
+        return version_match.group(1)
+    raise RuntimeError("Unable to find version string.")
+
+
+project_version = find_version(ospjoin(here, 'fast-namelist', '__init__.py'))
```

Comment:
this must be fastnml because it's the name of the directory with the init py file.